File: Project-TCL/Project/LDBaseModule/NetRequest/LDSocket/Const/ErrorCode.m
//
//  ErrorCode.m
//  TestSocket
//
//  Created by lingda on 2018/10/10.
//  Copyright © 2018年 TCL-MAC. All rights reserved.
//

#import "ErrorCode.h"

NSString * getErrorDescription(NSString * errorCode) {
    switch (errorCode.integerValue) {
        case success:
            return @"成功";
            break;
        case sysError:
            return @"系统错误";
            break;
        case notFoundUser:
            return @"没有找到此用户";
            break;
        default:
            return @"未知错误";
            break;
    }
}

File: TestSocket/TestSocket/CommonTool/LDSocket/Const/ErrorCode.swift
//
//  File.swift
//  TestSocket
//
//  Created by lingda on 2018/9/30.
//  Copyright © 2018年 TCL-MAC. All rights reserved.
//

import Foundation

enum ErrorCode : Int{
    case success = 0
    case systemError = 10000
    
    func description() -> String {
        switch self {
        case .success:
            return "成功"
        case .systemError:
            return "系统错误"
        default:
            break
        }
    }
}
